CHANGELOG.md

```diff
@@ -0,0 +1,81 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+### Added
+### Changed
+### Removed
+
+## 0.16.0
```

Comment:
We can add the date to get a quick overview of the release changes:
```suggestion
## [0.16.0] - 30/08/2021
```

or other time formats of course.

Reply:
I did not add the date as it can add a dependency, and thinking what if the release slips after pushing the changes. :) Folks can refer to the one on the Releases page for the date?

Reply:
> what if the release slips after pushing the changes. :)

To me, the `CHANGELOG` **is** the release. If we decide to release today, we can pin the date in the changelog, then tag the release.
If tomorrow, we have changes to add, they will go under the `[unreleased]` section.

Or I missed something :)